apps/sms/views/inbox/js/inbox.js

```diff
@@ -973,7 +984,21 @@ var InboxView = {
     }
   },
 
-  onDraftSaved: function inbox_onDraftSaved() {
+  onDraftDeleted: function inbox_onDraftDeleted(draft) {
```

Comment:
I think it might be possible to trigger the thread ui update twice while we trying to delete threads that contains drafts in inbox.  Generally we only want the thread update because of changes in conversation because inbox with update thread right in inbox view. Maybe we could set an option while deleting draft in conversation view and only emit delete event for this case?